app/controllers/volunteer_ops_controller.rb

```diff
@@ -5,7 +5,19 @@ class VolunteerOpsController < ApplicationController
   def index
     @volunteer_ops = VolunteerOp.order_by_most_recent
     @organisations = Organisation.where(id: @volunteer_ops.select(:organisation_id))
-    @markers = build_map_markers(@organisations)
+    harrow_markers = build_map_markers(@organisations)
+    doit_markers = '[]'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.